### ISSUE TITLE
WebdriverIO misses type support to Devtools service

### ISSUE THREAD

**Issue by lh104** [webdriverio/webdriverio#5352]
error TS2339: Property 'cdp' does not exist on type 'BrowserObject' when calling browser.cdp('Profiler', 'enable') in Typescript.

**Comment by christian-bromann** [comment]
I suggest to add a type definition to the [@wdio/devtools-service](https://github.com/webdriverio/webdriverio/tree/master/packages/wdio-devtools-service) similar to how we have these in other services and then allow the user to add them to their TypeScript config like:

```js
{
    "compilerOptions": {
        "baseUrl": ".",
        ...
        "types": ["node", "@wdio/sync", "expect-webdriverio", "@wdio/devtools-service"]
    },
    ...
}
```

**Comment by christian-bromann** [comment]
Fixed in #5361
